tests/unit/h/conftest.py

```diff
@@ -67,6 +67,12 @@
 from tests.common import factories as common_factories
 
 
+@pytest.fixture
+def es_client(es_client, clear_search_index):
+    yield es_client
+    clear_search_index()
```

Comment:
This preserves the previous behaviour of the _unit_ tests: they always clear the search index after every test.

Reply:
Yeah. This is preserving existing behaviour, but I agree it seems more robust to clear the search index before a test rather than after. For some reason cleaning up _after_ a test seems to be the common and recommended approach. This seems to be a harmless change to make so I've sent a follow-up PR for it: https://github.com/hypothesis/h/pull/9653